src/axom/core/numerics/Determinants.hpp

```diff
@@ -82,26 +86,37 @@ inline real determinant( const real& a00,  const real& a01,  const real& a02,
  * \param [in] a33 matrix element
  * \return det the determinant of the 4x4 matrix
  */
-template < typename real >
-inline real determinant(
-  const real& a00, const real& a01, const real& a02, const real& a03,
-  const real& a10, const real& a11, const real& a12, const real& a13,
-  const real& a20, const real& a21, const real& a22, const real& a23,
-  const real& a30, const real& a31, const real& a32, const real& a33  )
+template <typename real>
+inline real determinant(const real& a00,
+                        const real& a01,
```

Comment:
The main thing clang-format does not provide (as far as I can tell) is the ability to cluster parameters or arguments in a semantically meaningful way.

This is particularly evident for calls to ``axom::determinant()`` where we'd want to cluster the parameters by matrix row/col,

Alternative options would try to "BinPack" as many arguments as would fit in a line.  
My opinion is that when the arguments do not all fit on a single line, it is cleaner for them all to be on separate lines.

Reply:
This makes me sad, that clang-format can't cluster args.  In my opinion, though, declaration, definition, and calls to `determinant` are some of the vanishingly rare cases where we would want this clustering feature.

Reply:
I disabled formatting over most of the calls to ``axom::determinant()`` via ``clang-format off``/``clang-format on``